onnx/test/reference_evaluator_backend_test.py

```diff
@@ -51,11 +51,13 @@
 from onnx.reference.ops.op_cast import cast_to
 
 # Number of tests expected to pass without raising an exception.
-MIN_PASSING_TESTS = 1230
+MIN_PASSING_TESTS = 1229
 
 # Update this list if one new operator does not have any implementation.
 SKIP_TESTS = {
     # mismatches
+    # expected is [0] when output is 0.
+    "test__pytorch_operator_operator_index",
```

Comment:
Just curious: is this test error captured because of the new shape check below (L322-330)? If we are confident it is incorrect, perhaps we can update/remove it.